src/main/java/com/parsing/parsers/parserLot/ParserLotService.java

```diff
@@ -0,0 +1,106 @@
+package com.parsing.parsers.parserLot;
+
+import com.parsing.model.LotResult;
+import com.parsing.model.Status;
+import com.parsing.repository.LotResultRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@Slf4j
+@Service
+@Transactional(readOnly = true)
+public class ParserLotService {
+
+    private final LotResultRepository lotRepository;
+
+    public void parsering(List<String> urls) throws IOException {
+        for (String url : urls) {
+            Document document = Jsoup.connect(url).get();
+            String source = document.getElementsByClass("tender-date padding-left-more").toString();
+            Pattern pattern = Pattern.compile("\\d{8}");
+            Matcher matcher = pattern.matcher(source);
+            int start = 0;
+            int end = 0;
+            int count = 0;
+            while (matcher.find()) {
+                count++;
+                start = matcher.start();
+                end = matcher.end();
+
+            }
+            String dk = source.substring(start,end);
+            String uri = url;
+            //parse pdfLink
```

Comment:
Divide this method for 3 or more private methods with distinctive names that explain what each of them do